src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/ScriptActionsOnRunningCluster.cs

```diff
@@ -176,48 +176,6 @@ public void TestScriptActionsAPIOnNonExistentCluster()
             }
         }
 
-        [Fact]
-        public void TestRunningScriptActionsOnDisabledSub()
```

Comment:
The test was added originally when HDInsight Script Actions had not GA'ed but was in preview. The test checked that Script Actions were disabled for subscriptions that were not whitelisted. Since this feature is globally enabled, whitelisting is no longer required and the test is not valid anymore.